Makefile

```diff
@@ -36,7 +36,7 @@ image-push:
 
 # Can be used to compile docs from current (HEAD) version
 dev:
-	docker run -it --rm --volume $(CURDIR):/antora --entrypoint /antora/dev-docker-entryfile.sh antora/antora
+	docker run -it --rm --volume $(CURDIR):/antora --entrypoint /antora/dev-docker-entryfile.sh antora/antora:2.3.4
```

Comment:
It's good you've added this, but please move this bit to the main version branches (both 2022.9 and 2023.3)

Reply:
you mean like open PRs on both those versions or force push?